app/.gitignore

```diff
@@ -1 +0,0 @@
-/build
```

Comment:
Did you do this on purpose? Without this .gitignore a lot of temporary build files show up...